### DISCUSSION TITLE
Create Efficient Load Testing With Playwright and Artillery!

### DISCUSSION THREAD

**Discussion by amrsa1** [artilleryio/artillery#2000] [Show and Tell]
Hey friends, I have a new blog post that can assist you in creating your first load test scenarios more easily. In the blog, I'll show you how to use playwright and artillery to make it interesting and require less effort. Additionally, you'll discover some cool tips and tricks. I hope you find it helpful for your load testing!

https://lnkd.in/eiVkmG2W

**Comment by hassy** [comment]
Great blog post @amrsa1!
